src/player.cpp

```diff
@@ -8690,7 +8691,6 @@ hint_rating player::rate_action_unload( const item &it ) const
     if( it.has_flag( "NO_UNLOAD" ) ) {
         return HINT_CANT;
     }
-
```

Comment:
Why'd you remove this space?

Reply:
...I have no idea, that must of happened when I astyled.

Reply:
Looks like putting it back cause jenkins to flag an astyle regression, too.